test/Nancy.Swagger.Tests/SwaggerTypeMappingTests.cs

```diff
@@ -0,0 +1,37 @@
+using Should;
+using Xunit;
+
+namespace Nancy.Swagger.Tests
+{
+    public class SwaggerTypeMappingTests
+    {
+        [Fact]
+        public void ShouldAdd_AndGet_TypeMappings()
+        {
+            SwaggerTypeMapping.ResetMappedTypes();

```

Comment:
Is there a some type of annotation you can use to define a function that should run before (or after) each test?

Reply:
Not with XUnit unfortunately

Reply:
Looks like that was intentional: https://lostechies.com/jimmybogard/2013/09/26/test-styles-and-avoiding-setupteardown/

Reply:
Maybe we can migrate to nunit